apiserver/facades/client/cloud/cloud.go

```diff
@@ -445,16 +424,6 @@ func (api *CloudAPI) AddCredentials(ctx context.Context, args params.TaggedCrede
 	return results, nil
 }
 
-// CheckCredentialsModels validates supplied cloud credentials' content against
-// models that currently use these credentials.
-// If there are any models that are using a credential and these models or their
-// cloud instances are not going to be accessible with corresponding credential,
-// there will be detailed validation errors per model.
-// There's no Juju API client which uses this, but JAAS does,
-func (api *CloudAPI) CheckCredentialsModels(ctx context.Context, args params.TaggedCredentials) (params.UpdateCredentialResults, error) {
```

Comment:
So we confirmed this is no longer used yeah?

Reply:
Yeah, JAAS guys are aware  - it was just added for them as far as I am aware. It was the old way to update a model credential.